examples/awp_azure_onboarding/README.md

```diff
@@ -0,0 +1,111 @@
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/dome9/dome9-sdk-go/dome9"
+	"github.com/dome9/dome9-sdk-go/services/awp_azure_onboarding"
+)
+
+func main() {
+	// Pass accessID, secretKey, rawUrl, or set environment variables
+	config, _ := dome9.NewConfig("Access-ID", "Secret-Key", "https://api.dome9.com/v2/")
+	srv := awp_azure_onboarding.New(config)
+
+	// Get cloud account ID
+	cloudAccountId, _, err := srv.GetCloudAccountId("Azure subscription ID or Cloudguard Account ID")
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("Cloud Account ID: %s\n", cloudAccountId)
+	
+	// Define the request
+	awpAzureOnboardingDataRequest := awp_azure_onboarding.CreateAWPOnboardingDataRequest{
+		CentralizedId:     "string", // optional 
+	}
+
+	// Get awp azure onboarding data
+	appAzureOnboardingDataResponse, _, err := srv.Get(cloudAccountId, awpAzureOnboardingDataRequest)
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Printf("Get AWP Azure Onboarding Data response type: %T\n Content: %+v", appAzureOnboardingDataResponse, appAzureOnboardingDataResponse)
+
+
+	// Define the request
+	awpAzureOnboardingRequest := awp_azure_onboarding.CreateAWPOnboardingRequest{
```

Comment:
Can you define multiple requests for each scan mode ? i remeber based on the scan mode, there is a need to adjust : 
ScanMachineIntervalInHours:   4,
MaxConcurrenceScansPerRegion: 1,